src/module/item/base.ts

```diff
@@ -585,6 +585,18 @@ class ItemPF2e extends Item<ActorPF2e> {
         if (changed.data?.description?.value === null) {
             changed.data.description.value = "";
         }
+
+        // Run preUpdateItem rule element callbacks
+        type WithPreUpdate = RuleElementPF2e & { preUpdate: NonNullable<RuleElementPF2e["preUpdate"]> };
+        const rules = this.rules.filter((r): r is WithPreUpdate => !!r.preUpdate);
+        if (rules.length > 0) {
+            const clone = this.clone(changed, { keepId: true });
+            this.data.flags.pf2e.rollOptions = clone.data.flags.pf2e.rollOptions;
```

Comment:
What's this part for?

Reply:
Ah true, don't need to clone the rollOptions for items, that was lifted from the implementation in `ActorPF2e _preUpdate`